app/models.py

```diff
@@ -17,13 +17,9 @@
 class User(AbstractUser):
     """
     Custom user model to use in the future.
-    Email is required to create an User.
     """
-
-    email = models.EmailField(null=False, unique=True, blank=False)
-    display_name = models.CharField(max_length=150, null=False, blank=True)
-
-    EMAIL_FIELD = "email"
+    bio = models.TextField(max_length=255, blank=True)
+    photo = models.ImageField(upload_to='avatars', blank=True)
```

Comment:
You've added two new fields but I don't see them used anywhere (apart from admin panel).

Reply:
I am now making big changes to your project, and you will be shadowed by it soon. Don't worry

Reply:
Cool, looking forward to them.